### DISCUSSION TITLE
Updating current depending on actual capacity during ageing

### DISCUSSION THREAD

**Discussion by RSPdatascience** [pybamm-team/PyBaMM#4448] [Q&A]
Hi,

During an ageing experiment, how can I update the C-rate after every loop, based on actual capacity rather than nominal capacity ?

I mean something like this:

parameter_values = pybamm.ParameterValues("Chen2020")

model = pybamm.lithium_ion.SPM(
    options={
        "thermal": "lumped",            
        "SEI": "ec reaction limited",
        "SEI film resistance": "distributed",
    }
)

def i_fun_capacity(variables):
    capacity = ...................
    return capacity * 1  

experiment = pybamm.Experiment(
    [
        (
            pybamm.step.CustomStepExplicit(i_fun_capacity, termination="4.2V", direction="charge"),
            "Hold at 4.2V until C/50",  # It would be nice to replace this C by actual capacity too
            pybamm.step.CustomStepExplicit(i_fun_capacity, termination="2.5V", direction="discharge"),            
        )
    ]
    * 100,
    termination="70% capacity", 
)

sim = pybamm.Simulation(model, experiment=experiment, parameter_values=parameter_values)
sol = sim.solve()

**Comment by Florana1** [comment]
Hi, l am also wondering how to get the current remaining capacity. l'm not sure if we could use ['Total lithium capacity [A.h]'] instead. Because otherwise we need to do a full charge-dischagre to determin the current capacity of the battery.
